apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -724,59 +733,15 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
                 return (
                     <Segment basic >
                         <h5 className=" text-center"> { t(translateKey + "modals.scan.heading") }</h5>
-                        <Segment textAlign="center" basic className="qr-code">
-                            { qrCode
-                                ? <QRCode value={ qrCode } data-testid={ `${ testId }-view-modals-scan-qrcode` }/>
-                                : null
-                            }
-                        </Segment>
                         <div className="ml-3 mr-3">
-                            <Message className="display-flex" size="small" warning>
-                                <Icon name="warning sign" color="orange" corner />
-                                <Message.Content className="tiny">
-                                    { t(commonConfig.accountSecurityPage.mfa.totp.regenerateWarning) }
-                                </Message.Content>
-                            </Message>
-                            {
-                                commonConfig.accountSecurityPage.mfa.totp.showRegenerateConfirmation
```

Comment:
Let's remove the `showRegenerateConfirmation` from config files as well. With the improved UI this config won't be used.
https://github.com/wso2/identity-apps/blob/f5b6df60ae32bd48e851d7173124793f780979e7/apps/myaccount/src/extensions/configs/common.ts#L39
After the unification the config is always set to `false`. Therefore no point of keeping it at all.